src/components/ProductImages.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="prd-img">
+      <img src="@/assets/images/image-product-1.jpg" alt="product--image" id="1">
+      <img src="@/assets/images/image-product-2.jpg" alt="product-image" id="2">
+      <img src="@/assets/images/image-product-3.jpg" alt="product-image" id="3">
+      <img src="@/assets/images/image-product-4.jpg" alt="product-image" id="4">
+  </div>
+</template>
+
+<script>
+
+export default {
+
+}
+</script>
+
+
+<style scoped>
+.prd-img{
```

Comment:
[Request] Pls use verbose class names for your CSS

Reply:
noted